<a href="https://colab.research.google.com/github/Cubito918/data-analytics-portfolio/blob/main/ventas_analysis.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [5]:
import pandas as pd

# Cargar dataset desde GitHub

url = "https://raw.githubusercontent.com/Cubito918/data-analytics-portfolio/refs/heads/main/ventas-analysis/ventas.csv"

df = pd.read_csv(url)
df.head()

# Total Vendido por Producto
ventas_producto = df.groupby("producto")["monto"].sum()
print(ventas_producto)

#Total Vendido por Mes
ventas_mes = df.groupby("fecha")["monto"].sum()
print(ventas_mes)

# Producto "TOP" (más vendido)
producto_top = ventas_producto.idxmax()
print(producto_top)

#Cliente "TOP" (el que más gastó)
cliente_top = df.groupby("id_cliente")["monto"].sum().idxmax()
print(cliente_top)

producto
Laptop     29000
Monitor     9000
Mouse       2000
Teclado     1500
Name: monto, dtype: int64
fecha
2025-01     7000
2025-02    10000
2025-03     9500
2025-04    15000
Name: monto, dtype: int64
Laptop
105


## 📌 Insights

- Laptop es el producto más rentable.
- Abril registra el mayor ingreso mensual.
- El cliente 105 es un cliente de alto valor.